content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -213,7 +213,7 @@ new `PersistentVolume` is never created to satisfy the claim. Instead, an existi
 
 You can only resize volumes containing a file system if the file system is XFS, Ext3, or Ext4.
 
-When a volume contains a file system, the file system is only resized when a new Pod is started using
+When a volume contains a file system, the file system is only resized when a new Pod has started using
```

Comment:
I think this behavior is changing in 1.12. cc @gnufied

Reply:
Not merged yet -  https://github.com/kubernetes/kubernetes/pull/67608 needs approval from @saad-ali or something.